Makefile

```diff
@@ -26,7 +26,7 @@ GID = $(shell id -g)
 
 # The below files are needed and can be downloaded from the internet
 testdata_objects = testdata/vmlinux testdata/root-drive.img testdata/firecracker testdata/jailer
-firecracker_version = v0.23.0
```

Comment:
It would be better to split this PR into two. The first one is just upgrading Makefile, client/swagger.yaml and auto-generated files to support Firecracker 0.24.0. The second one is supporting ballooning through Machine.

Reply:
I can't split well-enough into commits, so I create a new branch and make a new PR. [Link to it](https://github.com/firecracker-microvm/firecracker-go-sdk/pull/317)
I will clean this development branch squashing some of the commits, sorry for the mess.